components/CharacterCard.vue

```diff
@@ -0,0 +1,50 @@
+<script lang="ts" setup>
+
+defineProps({
+  character: {
+    type: Object,
+    required: true
+  }
+});
+
+const bgByCharacterGender = (gender) => {
+  let bg;
```

Comment:
you could avoid assigning value to the `bg` variable, just return the value if condition is true

```
switch (gender) {
  case "male": {
    return "bg-[#008bff]";
  }
```